cmd/infra/aws/iam.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam/iamiface"
 )
 
+func DefaultIAMName(infraID string) string {
+	return "hypershift-default-" + infraID
```

Comment:
It's probably better to start the name with infraID so we follow the pattern of the rest of OCP (for when we leak stuff in CI)

Reply:
Fixed